tests/Suggestor/Suggestor.test.ts

```diff
@@ -206,8 +206,7 @@ describe.each([
     it('offers generic due date completions', () => {
         // Arrange
         const line = `- [ ] some task ${dueDateSymbol}`;
-        const suggestions = shouldStartWithSuggestionsContaining(line, ['today', 'tomorrow']);
-        expect(suggestions.length).toEqual(6);
```

Comment:
Deleted this Line as this was not expecting the new Default (20), which in my Eyes is not an Useful Test.

Previously, it Expected just the Old Default (6), which does not validate the Description 'offers generic due date completions'.

Reply:
Ok, but then please add a new test somewhere of the new default value.

Reply:
I will add a test for this.

Reply:
> I will add a test for this.

Done.